client/src/components/composite/SuccessfulBooking/SuccessfulBooking.tsx

```diff
@@ -0,0 +1,37 @@
+import FullPageBookLodgeImage from "components/generic/FullPageBookLodgeImage/FullPageBookLodgeImage"
+
+type SuccessfulBookingProps = {
+  startDate: string | null
```

Comment:
prefer the optional tag, as that handles both null and undefined
```suggestion
  startDate?: string
```